jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -19,6 +19,7 @@ def buildExclusions = [
   [ /centos[67]-(arm)?(64|32)-gcc6/,  anyType,     lt(10)  ],
   [ /centos6-32-gcc6/,                releaseType, gte(10) ], // 32-bit linux for <10 only
   [ /^centos7-64/,                    releaseType, lt(12)  ],
+  [ /^centos7-ppcle/,                 anyType,     lt(12)  ],
```

Comment:
OK, the new machines are labelled `centos7-ppcle`, this excludes them when node lt 12.